homeassistant/components/fujitsu_fglair/__init__.py

```diff
@@ -47,3 +49,24 @@ async def async_unload_entry(hass: HomeAssistant, entry: FGLairConfigEntry) -> b
         await entry.runtime_data.api.async_sign_out()
 
     return unload_ok
+
+
+async def async_migrate_entry(hass: HomeAssistant, entry: FGLairConfigEntry) -> bool:
+    """Migrate old entry."""
+    if entry.version > 1:
+        return False
+
+    if entry.version == 1:
+        new_data = {**entry.data}
+        if entry.minor_version < 2:
+            is_europe = new_data.pop(CONF_EUROPE, False)
```

Comment:
Let's not pop values. We do a minor version migration so that if people revert back to a release before this, everything keeps working. So by removing the value we make the entry invalid